### PR TITLE
Fix MemSQL test failure: testRenameTableAcrossSchema

### PR DESCRIPTION
**Pull request by kokosing**

### CODE REVIEW

File: plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlConnectorTest.java [hunk 1]
```diff
@@ -89,6 +89,9 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_ARRAY:
                 return false;
 
+            case SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS:
```
Comment (nineinchnick) [inline plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlConnectorTest.java:92]: This should also be added to `TestMemSqlLatestConnectorSmokeTest`, sorry I missed that in my commit in #7333

**Review by findepi — APPROVED** [review]

**Comment by losipiuk** [comment]
Not suppoted indeed. But fails with wrong exception messsage.
You need to add if to `MemSqlClient`

**Comment by nineinchnick** [comment]
> Not suppoted indeed. But fails with wrong exception messsage.
> You need to add if to `MemSqlClient`

I can't push here but I've added fixes in my fork: https://github.com/trinodb/trino/compare/master...nineinchnick:jwas/memsql-rename-across-schema and figured out how to run these tests locally and they pass. @losipiuk can you rebase this branch?

**Comment by nineinchnick** [comment]
> Not suppoted indeed. But fails with wrong exception messsage.
> You need to add if to `MemSqlClient`

I've made the required changes in my fork (can't push to this branch): https://github.com/trinodb/trino/compare/master...nineinchnick:jwas/memsql-rename-across-schema

I figured out how to run tests locally and they pass.

**Comment by findepi** [comment]
> I've made the required changes in my fork (can't push to this branch): [master...nineinchnick:jwas/memsql-rename-across-schema](https://github.com/trinodb/trino/compare/master...nineinchnick:jwas/memsql-rename-across-schema)

force-pushed here

(fixed one codestyle and changed commit message, sorry)

**Review by losipiuk — APPROVED** [review]

**Review by nineinchnick — APPROVED** [review]

**Comment by findepi** [comment]
memsql tests completed

**Pull request merged**